river/tree/mondrian/mondrian_tree_nodes.py

```diff
@@ -0,0 +1,474 @@
+import typing
+
+from abc import ABC, abstractmethod
+
+from river.utils.math import log_sum_2_exp
+
+from math import log
```

Comment:
I suggest importing math and using `math.log`

Reply:
Solved in latest commit.